src/System.Private.ServiceModel/tests/Common/Infrastructure/OsID.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Infrastructure.Common
+{
+    [Flags]
+    public enum OSID
+    {
+        None = 0,
+
+        Windows_7 = 1,
+        Windows_8_1 = 2,
+        Windows_10 = 4,
+        WindowsPhone = 8,
+        Windows_Nano = 16,
+
+        CentOS_7_1 = 32,
+        AnyCentOS = CentOS_7_1,
+
+        Debian_8 = 64,
+        AnyDebian = Debian_8,
+
+        Fedora_23 = 128,
+        AnyFedora = Fedora_23,
+
+        OpenSUSE_13_2 = 256,
+        AnyOpenSUSE = OpenSUSE_13_2,
+
+        OSX_10_11 = 512,
+        AnyOSX = OSX_10_11,
+
+        RHEL_7_2 = 1024,
+        AnyRHEL = RHEL_7_2,
+
+        Ubuntu_14_04 = 2048,
+        Ubuntu_16_04 = 4096,
+        AnyUbuntu = Ubuntu_14_04 | Ubuntu_16_04,
+
+        Unknown = 8192,     // were unable to determine
```

Comment:
>Unknown = 8192 [](start = 8, length = 14)

Unknown shouldn't be the next number in sequence. It's not possible to have no OS, so maybe Unknown should have a value of 0 so that Unknown is the opposite of Any.